kube-audit/kube-audit-policy.yaml

```diff
@@ -0,0 +1,105 @@
+# This policy is used by the audit webhook to collect and forward any events that are passed
+# through your Kubernetes API server to IBM Log Analysis with LogDNA. Fore more information,
+# see the docs: https://cloud.ibm.com/docs/containers?topic=containers-health#webhook_logdna
+apiVersion: audit.k8s.io/v1
+kind: Policy
+rules:
+  - level: None
+    userGroups: ["system:serviceaccounts:kube-system"]
+  - level: None
+    # Ingress controller reads `configmaps/ingress-uid` through the unsecured port.
+    users: ["system:unsecured"]
+    namespaces: ["kube-system"]
+    verbs: ["get"]
+    resources:
+      - group: "" # core
```

Comment:
what does the `# core` comment mean?

Reply:
hmm good q - i might remove some of the comments bc they mean things intermally but not useful for external users